wiki/Tournaments/SSR/1/en.md

```diff
@@ -0,0 +1,955 @@
+11. Any attempts to manipulate bracket outcomes or future matchups may result in the immediate disqualification of the player from the competition. Examples of manipulative behavior include, but are not limited to:
+    - Deliberately playing poorly to lose a game or match.
+    - Colluding with other players to force a desired matchup.
+    - Forfeiting bracket matches without providing an adequate reasoning, or with the intention of getting a more favourable matchup.
+12. All players and staff must be treated with respect. Instructions from the referees and the tournament organisers are to be followed. Decisions labelled as final are not to be objected.
+13. The multiplayer chatrooms underlie the [osu! community rules](/wiki/Rules). All chat rules apply to the multiplayer chatrooms where the matches will take place.
+    - Breaking the chat rules may result in a silence. Silenced players cannot participate in multiplayer matches for the duration of the punishment and, as a result, can have their match forfeited.
+14. Penalties for violating the tournament rules include, but are not limited to:
+    - Removal of pick rights for a map or an entire match.
+    - Point deductions.
+    - Declaring the match as forfeited, or as a win by default for the other player.
+    - Disqualification from the entire tournament.
+    - Disqualification from the current and future official tournaments, until appealed.
+15. Referees may allow, at their discretion, lower or higher tolerances for timers.
+16. The tournament organisers may request liveplays or recordings of individual players at any point in the tournament without prior warning.
+17. The tournament organisers will be responsible for receiving and investigating any tournament-related complaints.
+18. The tournament organisers reserve the right to modify these rules at any moment. Any such changes will be announced in advance.
+
+### Registration
+
+1. Every user interested in playing will be be required to sign up through the Player Registrations form.
+2. To ensure valid registrations, every qualifying participant will be manually checked through the Tournament screening process.
+3. Participants will be briefed on further steps via an announcement message at the end of the Player Registrations form.
+4. Tournament staff members are not allowed to play in the tournament, with the exception of streamers, commentators, and graphic designers. Eliminated players may be enlisted as mappool playtesters and/or replay creators. Note that per conventional Officially supported tournament rules, eliminated players may be enlisted as referees, however this role is not open for such individuals at the moment.
+   - The tournament managers must ensure that such staff members do not have access to priviledged information, e.g. Qualifiers score information, mappool information before the mappool showcase, etc. They are to be treated as regular players whenever possible.
```

Comment:
More correct to add the example as an in-lined parenthesis + corrected "privileged"
```suggestion
   - The tournament managers must ensure that such staff members do not have access to privileged information (e.g. Qualifiers score information, mappool information before the mappool showcase). They are to be treated as regular players whenever possible.
```